### ISSUE TITLE
M30T FFC returns an error

### ISSUE THREAD

**Issue by eric-dk** [dji-sdk/Mobile-SDK-Android-V5#19]
Steps-to-reproduce:
With the correct `ComponentIndexType` and `CameraLensType.CAMERA_LENS_THERMAL` use `CameraKey.KeyThermalTriggerFFC.action(…)`.

Expected:
Camera performs flat-field correction with audible click.

Actual:
```
DJIError{errorCode='CORE_REQUEST_HANDLER_NOT_FOUND', innerCode='', description='null', hint='error code = -1'}
```

**Comment by DJI-William** [comment]
Have you set the KeyCameraVideoStreamSource as INFRARED_CAMERA firstly?
![image](https://user-images.githubusercontent.com/77274755/179167621-1825e9c8-854a-4053-98c2-baf74e235092.png)

**Comment by eric-dk** [comment]
All prerequisites done; same error.

**Comment by DJI-William** [comment]
Please have a look my testing video and it works.
Link：https://pan-sec.djicorp.com/s/QDkHTQt85f5tdnE
dji123

**Comment by DJI-William** [comment]
@eric-dk Is the problem solved in 5.0 version?
